__e2e__/init.test.ts

```diff
@@ -57,7 +57,7 @@ test('init fails if the directory already exists', () => {
 test('init should prompt for the project name', () => {
   const {stdout} = runCLI(DIR, ['init']);
 
-  expect(stdout).toContain('How would you like to name the app?');
+  // expect(stdout).toContain('How would you like to name the app?');
```

Comment:
Is this intended?

Reply:
I was just testing what happen when we're running prompt in CI